jobs/mobile-developer.md

```diff
@@ -0,0 +1,57 @@
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## How a good candidate looks
+
+- You have solid experience either in Android or iOS.
+- You wish to work with React Native, even if you have not much experience in it.
+- You have an opinion on how to architect the app and can compare MVP with MVVM.
+- You know how different components of mobile OSes work: push notifications, keystore & keychain, background tasks, GPS api, etc.
+- Have experience releasing apps to stores and understand the whole process from building to signing to publishing.
+- You value high-quality maintainable code: know how organize codebase the way when something changes in API we don't need to rewrite a lot of code across the app.
+- Have experience writing end-to-end and unit tests and maintain high coverage.
+- Your English skills allow you to feel comfortable communicating in it (verbally and written) the whole working day.
+- You know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication.
+- Learning process is an important part of your professional activities.
+- Teamwork is not an abstract word for you, but a set of principles and agreements you follow.
+
+
+## What you'll definitely need to work with us
+
+- Experience in writing native mobile apps for Android or iOS.
+- Experience in working with React Native or desire to learn it.
+- English knowledge to communicate with colleagues and clients.
+- Communication skills to collaborate remotely.
+- Independence and initiative - we won't control how you work, but we expect the work to be done.
+
+## About us
+
+We work remotely [#remote](https://github.com/datarockets/career#remote) and know how to do it well.
+
+We care about people [#people-first](https://github.com/datarockets/career#people-first) and projects [#work-as-a-team-with-clients](https://github.com/datarockets/career#work-as-a-team-with-clients).
+
+We build mobile and web apps for our clients. See [case studies](https://datarockets.com/case-studies/). Usually we work with clients for years and work as a single team.
+
+## Perks
+
+- 50% compensation for self-development activities: courses, conferences, books, educational platforms, English, sports.
+- Remote work and flexible schedule (actually no schedule at all - you decide when you work).
+- 32 paid days off per year - you can use them for vacations, occasional days off, and public holidays, only workdays count.
+- Full coverage of sick leaves.
+- 200 USD per year for medical expenses.
+
+## How to apply 💌
+
+In order to apply send us a message to [join@datarockets.com](mailto:join@datarockets.com) with your CV. In the message tell us about you and your experience, share some examples of your code.
```

Comment:
@vshkl, @maryili, I like to ask candidates some questions they answer when they apply. Usually that helps me to understand a level of developer and provokes a lot of discussions on an interview. E.g. if a candidate tells me that they use MVVM architecture for code organization w/o detail I may ask many additional questions: where does a business logic go, how does communication with API look like and where code responsible for this lie, how do you make sure API-communication code is independent from UI code, etc.

What do you think if we ask some questions in advance? Questions I may think of:
- How would you organize you code so when a business logic changes we don't have to rewrite code related to UI? Can you share an example from your practice?
- How do you write tests: which practices do you use and which you avoid? What kind of tests do you write and why?

@vshkl, do you have better suggestion for questions?

Reply:
great idea, will add this!